### PR TITLE
Change color of time input bars

### PR DESCRIPTION
**Pull request by pratyush1712**
### Summary

Modified CSS to change color of the time input bars

### Trello Task
<blockquote class="trello-card"><a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;7YlGYReC&#x2F;401-focus-indicator-on-time-input-lacks-sufficient-color-contrast-at-1191-301-is-required">Focus indicator on time input lacks sufficient color contrast at 1.19:1 (3.0:1 is required)</a></blockquote>

### Test Plan
image

### CODE REVIEW

**Review by pratyush1712 — COMMENTED** [review]
i am not sure why the other files were committed without any changes but i don't think there would be any issues as there were no changes made

**Review by epicdragon44 — DISMISSED** [review]
As you may know, we can't have nice things in frontend-land. What that means in this context is that you took care of the color contrast on Webkit, but not on Mozilla-based browsers.

Note that here, in my screenshot on Firefox, the color contrast is still wrong.
Screenshot 2022-12-09 at 2 30 52 AM

Refer to [this StackOverflow thread for clarity on the use of browser-specific tags](https://stackoverflow.com/questions/18083056/what-are-moz-and-webkit)

Please address Firefox! It's a lonely browser :( it needs all the help it can get.

File: frontend/src/components/RequestRideModal/requestridemodal.module.css [hunk 1]
```diff
@@ -1,6 +1,28 @@
 .inputContainer {
   max-width: 45rem;
 }
+.input[type='time']::-webkit-datetime-edit-hour-field:focus,
+.input[type='time']::-webkit-datetime-edit-minute-field:focus,
+.input[type='time']::-webkit-datetime-edit-ampm-field:focus {
+  background-color: #3783f6;
+}
+
```
Comment (SGupta101) [inline frontend/src/components/RequestRideModal/requestridemodal.module.css:9]: these changes work on all browsers, good job with this PR

**Review by SGupta101 — APPROVED** [review]

**Review by SGupta101 — APPROVED** [review]
these changes work on all browsers, good job with this PR

**Pull request merged**